lib/ansible/modules/cloud/azure/azure_rm_subnet.py

```diff
@@ -63,9 +61,8 @@
     route_table:
         description:
             - The reference of the RouteTable resource.
-            - It can accept both a str or a dict.
-            - The str can be the name or resource id of the route table.
-            - The dict can contains C(name) and C(resource_group) of the route_table.
+            - Can be the name or resource ID of the route table.
+            - Can be a dict containing the I(name) and I(resource_group) of the I(route_table).
```

Comment:
Since this is the description of `route_table`, let's not make it `I(. . .)`.

```suggestion
            - Can be a dict containing the I(name) and I(resource_group) of the route table.
```